stacker/util.py

```diff
@@ -466,8 +466,13 @@ def __init__(self, stacker_cache_dir=None):
             stacker_cache_dir (string): Directory of stacker local cache.
                                         Default to $HOME/.stacker
         """
+        if os.name == 'nt':  # if windows
+            home_env_name = 'USERPROFILE'
+        else:
+            home_env_name = 'HOME'
         stacker_cache_dir = (stacker_cache_dir or
-                             os.path.join(os.environ['HOME'], '.stacker'))
```

Comment:
I think `os.path.expanduser` is what you want here.  It should be cross-platform ready, so you should be able to do:

```
os.path.expanduser("~/.stacker")
```

And I believe it should just work.